Gruntfile.js

```diff
@@ -1,16 +1,22 @@
 module.exports = function (grunt) {
     const srcCommon = [
-        'src/spine.js',
         'src/component/spine-component-system.js',
         'src/component/spine-component.js',
         'src/component/spine-component-data.js',
+        'contrib/semver/iife/semver.js',
         'src/spine-plugin.js'
     ];
     const src36 = [
-        'contrib/spine-ts/build/3.6/spine-core.js'
+        'contrib/spine-ts/build/3.6/spine-core.js',
+        'src/spine.js'
```

Comment:
You can move this back up to `srcCommon`?

Reply:
Completed in: 4aab6ec